modules/lofreq/1.0/lofreq.smk

```diff
@@ -67,7 +67,8 @@ rule _lofreq_run:
     threads:
         CFG["threads"]["lofreq"]
     resources:
-        mem_mb = op.retry(CFG["mem_mb"]["lofreq"])
+        mem_mb = CFG["mem_mb"]["lofreq"],
```

Comment:
Was this a bug?

Reply:
I don't think it is a bug. I think it did not work for all the samples as it was originally thought. I found that it will multiply whatever is in `mem_mb` by 1.5, and this will be used as a resource for the job scheduling. Some samples were completed successfully, but others - exited being killed by slurm for exceeding memory limit. On the other hand, the same samples were completed if I specify memory directly, without a function. Therefore, I changed it here.